charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -110,7 +110,7 @@ spec:
           - name: REACT_APP_MERLIN_API
             value: {{ include "common.set-value" (list .Values.ui.apiHost $globMerlinApi) | quote }}
           - name: REACT_APP_MLP_API
-            value: {{ include "common.set-value" (list .Values.ui.mlp.apiHost $globMlpApi) | quote }}
+            value: "{{ .Values.ui.mlp.apiHost }}"
```

Comment:
@ariefrahmansyah, should we instead just drop the `apiPrefix` from:
```
{{- $globMerlinApi := include "common.get-component-value" (list .Values.global "merlin" (list "vsPrefix" "apiPrefix"))}}
```

So `{{ .Values.ui.mlp.apiHost }}` doesn't have to be set explicitly?